testing/py-google-apputils/APKBUILD

```diff
@@ -22,8 +22,9 @@ build() {
 package() {
 	cd "$builddir"
 	python setup.py install --prefix=/usr --root="$pkgdir" || return 1
+	chmod o+r "$pkgdir"/usr/lib/python2.7/site-packages/google_apputils-"$pkgver"-py2.7.egg-info/* || return 1
```

Comment:
That’s okay, just please add a comment explaining that it’s a workaround for that issue and add link to the issue.